src/main/java/racingcar/moving/RandomMovingStrategy.java

```diff
@@ -0,0 +1,19 @@
+package racingcar.moving;
+
+import java.util.Random;
+
+public class RandomMovingStrategy implements MovingStrategy {
+
+    private static final int MIN_NUM = 0;
+    private static final int MAX_NUM = 9;
+    private static final int BASE_LINE_NUM = 4;
+
+    @Override
+    public boolean move() {
+        return random() >= BASE_LINE_NUM;
+    }
+
+    private int random() {
+        return new Random().nextInt(RandomMovingStrategy.MAX_NUM) + RandomMovingStrategy.MIN_NUM;
```

Comment:
```suggestion
        return new Random().nextInt(MAX_NUM) + MIN_NUM;
```
굳이 클래스까지 붙여준 이유가 있나요?

Reply:
그냥... 상수값에 대한 제 하나의 코딩 습관이라고 해야될까요....?ㅎㅎㅎ
동일한 클래스 내부에서 사용하는 거라면 간단하게 MAX_NUM 으로만 적어도 될 거 같아요!